scripts/opencontrail-install/contrail_install.rb

```diff
@@ -0,0 +1,324 @@
+    ssh_setup
+    @contrail_controller = IPSocket.getaddress(@controller_host)
+    error "Cannot resolve contrail-controller host" \
+        if @contrail_controller.empty?
+    Dir.chdir("#{@ws}")
+end
+
+# Download and extract contrail and thirdparty rpms
+def download_contrail_software
+    sh("wget -qO - https://github.com/rombie/opencontrail-netns/blob/master/provision/fedora/contrail-rpms.tar.xz?raw=true | tar Jx")
+    sh("wget -qO - https://github.com/rombie/opencontrail-netns/blob/master/provision/fedora/thirdparty.tar.xz?raw=true | tar Jx")
+end
+
+# Install third-party software from /cs-shared/builder/cache/centoslinux70/juno
+def install_thirdparty_software_controller
+    sh("yum -y remove java-1.8.0-openjdk java-1.8.0-openjdk-headless")
+
+    init_common
+    sh("yum -y install supervisor supervisord python-supervisor rabbitmq-server python-kazoo python-ncclient", true)
+
+    third_party_rpms = [
+    "#{@ws}/thirdparty/authbind-2.1.1-0.x86_64.rpm",
+    "#{@ws}/thirdparty/librdkafka1-0.8.5-2.0contrail0.el7.centos.x86_64.rpm",
+    "#{@ws}/thirdparty/librdkafka-devel-0.8.5-2.0contrail0.el7.centos.x86_64.rpm",
+    "#{@ws}/thirdparty/cassandra12-1.2.11-1.noarch.rpm",
+    "#{@ws}/thirdparty/kafka-2.9.2-0.8.2.0.0contrail0.el7.x86_64.rpm",
+    "#{@ws}/thirdparty/python-pycassa-1.10.0-0contrail.el7.noarch.rpm",
+    "#{@ws}/thirdparty/thrift-0.9.1-12.el7.x86_64.rpm",
+    "#{@ws}/thirdparty/python-thrift-0.9.1-12.el7.x86_64.rpm",
+    "#{@ws}/thirdparty/python-bitarray-0.8.0-0contrail.el7.x86_64.rpm",
+    "#{@ws}/thirdparty/python-jsonpickle-0.3.1-2.1.el7.noarch.rpm",
+    "#{@ws}/thirdparty/xmltodict-0.7.0-0contrail.el7.noarch.rpm",
+    "#{@ws}/thirdparty/python-amqp-1.4.5-1.el7.noarch.rpm",
+    "#{@ws}/thirdparty/python-geventhttpclient-1.0a-0contrail.el7.x86_64.rpm",
+    "#{@ws}/thirdparty/consistent_hash-1.0-0contrail0.el7.noarch.rpm",
+    "#{@ws}/thirdparty/python-kafka-python-0.9.2-0contrail0.el7.noarch.rpm",
+    "#{@ws}/thirdparty/redis-py-0.1-2contrail.el7.noarch.rpm",
+    "#{@ws}/thirdparty/ifmap-server-0.3.2-2contrail.el7.noarch.rpm",
+    "#{@ws}/thirdparty/hc-httpcore-4.1-1.jpp6.noarch.rpm",
+    "#{@ws}/thirdparty/zookeeper-3.4.3-1.el6.noarch.rpm",
+    "#{@ws}/thirdparty/bigtop-utils-0.6.0+243-1.cdh4.7.0.p0.17.el6.noarch.rpm",
+    "#{@ws}/thirdparty/python-keystone-2014.1.3-2.el7ost.noarch.rpm",
+    "#{@ws}/thirdparty/python-psutil-1.2.1-1.el7.x86_64.rpm",
+    "#{@ws}/thirdparty/java-1.7.0-openjdk-1.7.0.55-2.4.7.2.el7_0.x86_64.rpm",
+    "#{@ws}/thirdparty/java-1.7.0-openjdk-headless-1.7.0.55-2.4.7.2.el7_0.x86_64.rpm",
+    "#{@ws}/thirdparty/log4j-1.2.17-15.el7.noarch.rpm",
+
+    # "#{@ws}/thirdparty/python-psutil-0.6.1-3.el7.x86_64.rpm",
+    # "#{@ws}/thirdparty/python-keystone-2014.2.1-1.el7.centos.noarch.rpm",
+    ]
+    sh("yum -y install #{third_party_rpms.join(" ")}", true)
+end
+
+# Install contrail controller software
+def install_contrail_software_controller
+    contrail_rpms = [
```

Comment:
Does it make sense to have the contrail version be a variable ?
We probably would want to replace this to a well know stable version as soon as one is available that supports this platform.